benchmark/acmeair-standalone-corretto/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment to enable CDS
+#FROM eclipse-temurin:8u362-b09-jre-jammy as builder
+FROM amazoncorretto:8u362-alpine3.14-jre
+#COPY --from=builder /opt/java/openjdk/lib/classlist /usr/lib/jvm/java-8-openjdk-amd64/jre/lib
+#COPY --from=builder /bin/sh /bin/sh
+#USER root
```

Comment:
I'd clean up these comments

Reply:
done. Also bumped back tomcat to 9 - forgot that we are on spring 2 due to Java 8 and 10 is incompatible ;(